### DISCUSSION TITLE
Having DBeaver running on iPad

### DISCUSSION THREAD

**Discussion by galinyc1** [dbeaver/dbeaver#10300] [Q&A] [answered]
**Is your feature request related to a problem? Please describe.**
Not a problem but now that iPad supports mouse, more and more people are trying to make it their primary laptop

**Describe the solution you'd like**
DBeaver app in-app store to be at least be able to connect top-used databases (Oracle/MSSQL/PostgreSQL/MySQL)

**Describe alternatives you've considered**
None. there are other database applications for iPad but I would like to use this one.

**Additional context**
Have it in the app store will help someone to get some money for it :)

**Comment by tati-kru** [comment, marked as answer]
@gallicht we don't have plans to migrate DBeaver to iOS-native app. However, for your needs you can try CloudBeaver that is a web-application for working with databases. CloudBeaver is also open-source. It can be easily installed from Docker according to the [instruction](https://github.com/dbeaver/cloudbeaver/wiki/Run-Docker-Container).
